pom.xml

```diff
@@ -114,11 +110,6 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <configuration>
-          <systemPropertyVariables>
-            <useJvmChmod>${useJvmChmod}</useJvmChmod>
```

Comment:
The `useJvmChmod` system property is no longer used, so there is no point
in setting it. That is why I've removed it from the Surefire configuration.